aikiblog/templates/aikiblog/training_detail.html

```diff
@@ -2,48 +2,68 @@
+        <br><a href="{% url "user-detail" training.user.id %}">{{ training.user.first_name }} {{ training.user.last_name }}</a>
+            {% ifequal training.user.sex "M" %}
             był
             {% endifequal %}
-            {% ifequal object.user.sex "K" %}
+            {% ifequal training.user.sex "K" %}
             była
             {% endifequal %}
             na
-            {% ifequal object.type "T" %}
+            {% ifequal training.type "T" %}
             treningu
             {% endifequal %}
-            {% ifequal object.type "S" %}
+            {% ifequal training.type "S" %}
             stażu
             {% endifequal %}
-            <br> {{ object.date }} w {{ object.place }}
+            <br> {{ training.date }} w {{ training.place }}
         </div>
         <div id="post_body_training_detail"><br>
-        Sensei: {{ object.sensei }}<br><br>
+        <p><b>Sensei:</b> {{ training.sensei }}</p>
         <table>
         <tr>
-            <td id="table_header_cell">Technika</td>
+            <td width="140" id="table_header_cell">Technika&nbsp;&nbsp;&nbsp;</td>
             <td id="table_header_cell">Błędy</td>
         </tr>
-    {% for technique in object.techniques.all %}
-        <tr>
-        <td>{{ technique.stand.name }} {{ technique.attack.name }}&nbsp;&nbsp;&nbsp;<br> {{ technique.technique.name }}&nbsp;&nbsp;&nbsp;</td>
-        <td>{{ technique.mistakes }}</td>
-
-
+    {% for technique in training.techniques.all %}
+        <tr height="60">
+            <td valign="top" id="table_stand_cell">{{ technique.stand.name }}&nbsp;&nbsp;&nbsp;<br> {{ technique.attack.name }} {{ technique.technique.name }}&nbsp;&nbsp;&nbsp;</td>
+            <td valign="top" id="table_technique_cell">{{ technique.mistakes }}</td>
         </tr>
-
     {% endfor %}
-        </table><br><br>
-        <b>Notatki:</b><br>
-        {{ object.notes }}
-
+        </table>
+        <p><b>Notatki:</b></p>
+        {{ training.notes }}
+        <br><br>
+        <a name="comments"></a>
+        <p><b>Komentarze:</b></p>
+        <table>
+            {% for comment in training.trainingcomment_set.all %}
+                <tr>
+                    <td><img src="{{ STATIC_URL|cut:'static/' }}{{ comment.author.avatar }}" height=35></td>
+                    <td><b>{{ comment.author.first_name }} {{ comment.author.last_name }}</b> {{ comment.text }}<br>
+                    <font size=1>{{ comment.posted_date|date:"d E Y "}} o {{ comment.posted_date|date:" H:i " }}</font></td>
+                </tr>
+            {% empty %}
+                <tr><td></td>
+                <td>Nie dodano jeszcze komentarzy</td>
+                </tr>
+            {% endfor %}
+        {% if request.user.is_authenticated %}
+        <form method="post" class="form-horizontal" role="form">
+		{% csrf_token %}
+            <tr>
+                <td><img src="{{ STATIC_URL|cut:'static/' }}{{ request.user.avatar }}" height=35></td>

```

Comment:
Coś tu z tym cut jest nie tak. Załóż mi kiedyś na to zadanie, to się temu bliżej przyjrzę.